mta-multi-tenant.yaml

```diff
@@ -85,6 +85,7 @@ modules:
         url: ~{srv-url}
         forwardAuthToken: true
         strictSSL: true
+        timeout: 600000  # required for long running HTTP requests, e.g. large attachment uploads
```

Comment:
10 minutes might be a little bit too large, increases attack vector for [slow loris attacks](https://www.cloudflare.com/learning/ddos/ddos-attack-tools/slowloris/)

Reply:
what do you suggest ?

Reply:
Is the default enough (I suppose 2 minutes) or did you run into a concrete issue?

Reply:
The default is 30s, which causes issues during the upload of larger attachments. So, what should be the max. upload size we want to support out-of-the-box in the bookshop ?
E.g. I tested with very large attachments (up to 4GB) and it takes several minutes to upload them, depending on the network performance.

Reply:
4GB sound very unreasonable to me for a book cover 😆
I guess we should think about showcasing a size restriction (or providing such a validation as part of the attachment plugin out-of-the-box). For smaller-sized images 30s should be enough. I don't think increasing the timeout to 10min should be a good practice we want to showcase here.

Reply:
Removed as requested. From my point of view we should demonstrate how to configure this timeout, because I assume stakeholders will run into this issue.